app/src/main/java/woowacourse/payments/ui/screen/registration/CardRegistrationActivity.kt

```diff
@@ -0,0 +1,34 @@
+package woowacourse.payments.ui.screen.registration
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import woowacourse.payments.ui.model.CardUiModel
+import woowacourse.payments.ui.screen.cards.CardsActivity
+import woowacourse.payments.ui.theme.AndroidpaymentsTheme
+
+class CardRegistrationActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            AndroidpaymentsTheme {
+                CardRegistrationScreen(
+                    onBackPressed = { finish() },
+                    onCardRegistered = { registeredCard: CardUiModel ->
+                        val intent = CardsActivity.newIntent(this, registeredCard)
+                        setResult(RESULT_OK, intent)
+                        finish()
+                    },
```

Comment:
```suggestion
                    onCardRegistered = { registeredCard: CardUiModel ->
                        setResult(RESULT_OK, Intent().putExtra(EXTRA_NEW_CARD, paymentCard))
                        finish()
                    },
```

이렇게 `CardsActivity`에서 Intent를 생성하지 않아도 된다는 사실 알고 계셨나요 ??

`CardsActivity`가 `CardRegistractionActivity`를 호출했으므로 
`CardActivity`가 종료될 때 자동으로 `CardRegistractionActivity`로 결과가 간다고 하네요 !

따라서 데이터를 담을 `intent`만 필요하고 어떤 `Activity`로 갈지는 필요없다고 합니다 ( 신기... )

just 저도 신기해서 끄적끄적 적어 봤는데
**Extra 키 상수값**과 **받을 데이터를 받는 주체에서 관리**한다는 관점에서 채넛의 코드도 좋아 보이네요 👍🏻

Reply:
오호, 런처에 이동할 Activity를 지정해주고 콜백으로 값을 받을 수 있는 형태여서 가능한건가요 ?! 신기하네요 :)